index.d.ts

```diff
@@ -0,0 +1,29 @@
+interface UrlBuilderOptions {
+  allowUpscale: boolean,
+  backgroundColor: string,
+  borderRadius: string,
+  borderWidth: string,
+  circularMask: string,
+  color: string,
+  crop: string,
+  cropWithDifferentFocus: boolean,
+  disableWebP: boolean,
+  height: string,
+  flipHorizontally: boolean,
+  flipVertically: boolean,
+  forceDownload: boolean,
+  forceLossyCompression: boolean,
+  forceScaling: boolean,
+  imageFormat: string,
+  losslessCompression: string,
+  lossyCompressionLevel: string,
+  lossyCompressionVersion: string,
+  padding: string,
+  paddingColor: string,
+  rotate: string,
```

Comment:
```suggestion
  rotate: 90 | 180 | 270,
```

Reply:
Fixed in a1d5154eff961fc5d34a36ba7c9c06c0979ff2aa.